src/components/editor/Bindings/Backfill/index.tsx

```diff
@@ -0,0 +1,194 @@
+        [draftSpecs]
+    );
+
+    const evaluateServerDifferences = useCallback(
+        (increment: BooleanString) => {
+            if (bindingIndex === -1) {
+                return (
+                    (backfillAllBindings && increment === 'false') ||
+                    (!backfillAllBindings && increment === 'true')
+                );
+            }
+
+            if (currentCollection) {
+                return increment === 'true'
+                    ? !backfilledCollections.includes(currentCollection)
+                    : backfilledCollections.includes(currentCollection);
+            }
+
+            return false;
+        },
+        [
+            backfillAllBindings,
+            backfilledCollections,
+            bindingIndex,
+            currentCollection,
+        ]
+    );
+
+    const handleClick = useCallback(
+        (increment: BooleanString) => {
+            const serverUpdateRequired = evaluateServerDifferences(increment);
+
+            if (draftSpec && serverUpdateRequired) {
+                setFormState({ status: FormStatus.UPDATING });
+
+                const singleBindingUpdate =
+                    bindingIndex > -1 && currentCollection;
+
+                const bindingMetadata: BindingMetadata[] = singleBindingUpdate
+                    ? [{ collection: currentCollection, bindingIndex }]
+                    : [];
+
+                updateBackfillCounter(
+                    draftSpec,
+                    increment,
+                    bindingMetadata
+                ).then(
+                    () => {
+                        const targetCollection = singleBindingUpdate
+                            ? currentCollection
+                            : undefined;
+
+                        setBackfilledCollections(increment, targetCollection);
+                        setFormState({ status: FormStatus.UPDATED });
+                    },
+                    (error) => {
+                        setFormState({
+                            status: FormStatus.FAILED,
+                            error: {
+                                title: 'workflows.collectionSelector.manualBackfill.error.title',
+                                error,
+                            },
+                        });
+                    }
+                );
+            }
+        },
+        [
+            bindingIndex,
+            currentCollection,
+            draftSpec,
+            evaluateServerDifferences,
+            setBackfilledCollections,
+            setFormState,
+            updateBackfillCounter,
+        ]
+    );
+
+    return (
+        <Box sx={{ mt: 3 }}>
+            <Stack direction="row" spacing={1} sx={{ mb: 2 }}>
+                <Stack spacing={1}>
+                    <Typography
+                        variant={
+                            bindingIndex === -1 ? 'formSectionHeader' : 'h6'
+                        }
+                        sx={{ mr: 0.5 }}
```

Comment:
I do not think this `sx` is needed as it seems to not do anything.

Reply:
This property was set in an early permutation of the first, manual backfill PR. The button was located elsewhere and required a margin for alignment. Will remove.